docs/contributing/building.rst

```diff
@@ -153,6 +153,20 @@ or source it into your shell,
 
    . $BUILD_DIR/python_env.sh
 
+PyTorch
+=======
+
+To install PyTorch, follow the commands below. The first 2 lines install dependencies. The subsequent steps install PyTorch from source.
+
+.. code-block:: bash
+
+   conda env update --name katana-dev --file $SRC_DIR/external/katana/conda_recipe/pytorch_deps_environment.yml
```

Comment:
Is this path correct? I see ` conda_recipe/pytorch_deps_environment.yaml` but no `external/.../pytorch_deps_environment.yml`